app/Components/NavItem/NavItem.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import styles from "./NavItem.module.scss";
+
+type Props = {
+  visible: boolean;
+  navContent: string;
+  backgroundColor: string;
+};
+
+const NavItem = (props: Props) => {
+  const ItemStyle = {
+    backgroundColor: props.backgroundColor
+  };
+
+  const saleStylesRender = () => {
+    if (!props.visible) {
+      return styles.hidden;
+    } else {
+      return styles.sale;
+    }
+  };
+
+  return (
+    <div className={saleStylesRender()}>
+    {props.visible && (
+      <div className={styles.nav}>
+      <div className={styles.navContent}>
+        <div className={styles.navItem} style={ItemStyle}>
```

Comment:
u can directly pass the background color directly without external constant

Reply:
but the backgrand is different in pages so I made it with prop

Reply:
no ItemStyle itself is external u can pass it like style ={backgroundColor : props.background}